src/main/java/com/softserve/bookstore/controllers/BookController.java

```diff
@@ -25,10 +25,9 @@ public class BookController {
     private BookService bookService;
 
 
-    @GetMapping()
-    public ResponseEntity<List<Book>> findBook() throws IOException {
+    @GetMapping("/file")
+    public ResponseEntity<List<Book>> getAlBookFromTxT() throws IOException {
```

Comment:
better not use abbreviations to much
in this case getAllBooksFromFile is better, also check right spelling of words